mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs

```diff
@@ -29,6 +29,15 @@ public void Handler (int sn)
 			}
 		}
 
+
+		[Test]
+		public void GetPid ()
+		{
+			var currentPID = Mono.Unix.Native.Syscall.getpid();
+			Console.WriteLine ("CurrentPID is {0}", currentPID);
```

Comment:
please remove this unnecessary logging, we'll get the value in the assertion message anyway.